code/modules/projectiles/projectile.dm

```diff
@@ -920,13 +920,14 @@
 		finalize_hitscan_and_generate_tracers()
 	STOP_PROCESSING(SSprojectiles, src)
 	cleanup_beam_segments()
-	qdel(trajectory)
+	if(trajectory)
```

Comment:
You shouldn't need this check.

Reply:
trajectory can be null in some cases, tho I'll admit this is bleedover from some of the other stuff I'm doing. I can remove it if you'd like